tree/ntuple/v7/test/UnsplitLinkDef.h

```diff
@@ -0,0 +1,9 @@
+#ifdef __CLING__
+
+#pragma link C++ class ClassWithUnsplitMember + ;
+#pragma link C++ class CustomStreamer - ;
+#pragma link C++ options = rntuplesplit class CustomStreamerForceSplit - ;
+#pragma link C++ options = rntupleunsplit class CustomStreamerForceUnsplit + ;
```

Comment:
```suggestion
#pragma link C++ options=rntuple-split class CustomStreamerForceSplit-;
#pragma link C++ options=rntuple-unsplit class CustomStreamerForceUnsplit+;
```
or
```suggestion
#pragma link C++ options=rntupleSplit class CustomStreamerForceSplit-;
#pragma link C++ options=rntupleUnsplit class CustomStreamerForceUnsplit+;
```
or
```suggestion
#pragma link C++ options=rntupleSplit(true) class CustomStreamerForceSplit-;
#pragma link C++ options=rntupleSplit(false) class CustomStreamerForceUnsplit+;
```

Reply:
The last option has the advantage of being similar/same as the genreflex option and being 'even more clear' that it can be specified twice.